agent/src/main/java/org/openremote/agent/protocol/dmx/AbstractArtnetLightState.java

```diff
@@ -0,0 +1,19 @@
+package org.openremote.agent.protocol.dmx;
+
+import org.openremote.model.attribute.*;
+
+public abstract class AbstractArtnetLightState {
+
+    private int lightId;
+
+    public AbstractArtnetLightState(int lightId) {
+        this.lightId = lightId;
+    }
+    public int getLightId() {
+        return this.lightId;
+    }
+    public abstract Byte[] getValues();
+
+    public abstract void fromAttribute(AttributeEvent event, Attribute attribute);
```

Comment:
I only see 1 implementation of this abstract class....why not make it a proper class?